flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -114,6 +116,22 @@ private NetworkEnvironment(
 		this.isShutdown = false;
 	}
 
+	public static NetworkEnvironment fromConfiguration(
```

Comment:
It might be better to put this static factory at the end of this class, also for the following `create` method.